### DISCUSSION TITLE
Slow compiling

### DISCUSSION THREAD

**Discussion by tanhakabir** [microsoft/vscode-cpptools#8634] [Q&A]
https://github.com/microsoft/vscode/issues/139732

cc @Priyanshuwork

**Comment by Priyanshuwork** [comment]
Ok, thank you🙂

**Comment by bobbrow** [comment]
Hi @Priyanshuwork. Many factors affect the compilation of your code.  Our extension does not provide compilation services, but we're happy to help consult if you provide more information about your project.  I will convert this issue to a discussion as this is not likely a bug in the extension.

**Reply by Priyanshuwork** [reply]
Ok thank you😊

**Comment by sean-mcmanus** [comment]
What compiler is being used? How long is "too long"? How many files are being compiled?

**Reply by Priyanshuwork** [reply]
Compiler: Mingw 
Too longs means if I'm compiling on GDB it takes suppose 1 sec, on VS it's taking 5 sec.

Single file at a time it is compiling.
